jenkins/spark-premerge-build.sh

```diff
@@ -114,6 +116,8 @@ rapids_shuffle_smoke_test() {
 
 ci_2() {
     echo "Run premerge ci 2 testings..."
+    # export 'LC_ALL' to set locale with UTF-8 so regular expressions are enabled
+    export LC_ALL="en_US.UTF-8"
```

Comment:
same as above

Reply:
As far I know, to do this in `pytest.ini`, we would need this plugin for pytest installed in the CI env  (https://pypi.org/project/pytest-env/). Do we have this plugin available to us in CI?

Reply:
The underlying issue here is setting the locale, which is controlled by the shell. It's simple to override as an environment variable passed into the tests, but I don't know how to set that from Python, and use a mark for those tests.

Reply:
pytest.ini is supported in pytest by default as far as I know. We are using it in our pytest for a long time.

> override as an environment variable passed into the tests

This is good to me